### PR TITLE
Add SonarLint connected mode conf in IntelliJ for the plugin

### PR DESCRIPTION
**Pull request by sebastien-marichal**

### CODE REVIEW

File: .sonarlint/Plugin.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "sonarCloudOrganization": "sonarsource",
+    "projectKey": "org.sonarsource.dotnet:sonar-csharp"
```
Comment (pavel-mikula-sonarsource) [inline .sonarlint/Plugin.json:3]: Where did this came from? The project key is `org.sonarsource.dotnet:sonar-dotnet`

https://sonarcloud.io/project/overview?id=org.sonarsource.dotnet%3Asonar-dotnet

**Review by pavel-mikula-sonarsource — CHANGES_REQUESTED** [review]
The key seems wrong

File: .sonarlint/Plugin.json (hunk 1 above)
Comment (sebastien-marichal) [reply .sonarlint/Plugin.json]: I updated the content.
I am not sure how to test it.

I started Rider in a cleaned version of the repo (no prior .idea folder), and I had a pop-up to bind the project to `sonaranalyzer-dotnet`.
Is the manual binding still required with this configuration?

Comment (pavel-mikula-sonarsource) [reply .sonarlint/Plugin.json]: Yes, you should have a popup. You still need to provide token

**Pull request merged**